db_schema.md

```diff
@@ -86,3 +89,18 @@
 - user_id (integer, foreign key to user table)
 - user (auto reference to the User object that owns this Log)
 - icon (string, the icon to use for the bullet point in the notebook)
+
+### simulations_users (to track which simulations the user does and which users do which simulations)
+    - user_id
+    - sim_id
+
+### prompts_users (to track the answers the user gives to the prompts)
+    - prompt_id
+    - user_id
+    - answer (string)
+
+#### pages_users (to track which pages users go by and when)
+    - user_id
+    - page_id
+    - transition_count (integer, everytime a user visits a page, this is auto-incemented.)
```

Comment:
The transition count is an interesting idea. I think Dr. Peffer is really looking for a way to track the overall narrative, rather than just raw per-user pageload counts, you know? I guess if we add a timestamp to the pages_users thing that might work.